### PR TITLE
Create 2016-09-01 template deployment

### PR DESCRIPTION
**Pull request by chidmdxx**

### CODE REVIEW

**Comment by wenwu449** [comment]
Should only merge after the schema version supported in service.

File: schemas/2016-09-01/deploymentTemplate.json [hunk 1]
```diff
@@ -0,0 +1,525 @@
...
+                  { "$ref": "http://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/connections" },
+                  { "$ref": "http://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/variables" },
+                  { "$ref": "http://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/schedules" },
+                  { "$ref": "http://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/jobs" },
+                  { "$ref": "http://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/jobSchedules" },
+                  { "$ref": "http://schema.management.azure.com/schemas/2015-10-01/Microsoft.Media.json#/resourceDefinitions/mediaServices" },
+                  { "$ref": "http://schema.management.azure.com/schemas/2015-10-01/Microsoft.DataFactory.json#/resourceDefinitions/dataFactories" },
+                  { "$ref": "http://schema.management.azure.com/schemas/2015-10-01/Microsoft.DataFactory.json#/resourceDefinitions/linkedServices" },
```
Comment (bgold09) [inline schemas/2016-09-01/deploymentTemplate.json]: Data Factory schemas are being removed, see #235

File: schemas/2016-09-01/deploymentTemplate.json [hunk 2]
```diff
@@ -0,0 +1,525 @@
...
+                  { "$ref": "http://schema.management.azure.com/schemas/2015-10-01/Microsoft.DataFactory.json#/resourceDefinitions/datasets" },
```
Comment (bgold09) [inline schemas/2016-09-01/deploymentTemplate.json]: Data Factory schemas are being removed, see #235

**Comment by TianoMS** [comment]
@chidmdxx Do we still need this PR?

**Pull request closed without merging**